### DISCUSSION TITLE
Why is jupyter-book using sphinx-book-theme v0.1.4?

### DISCUSSION THREAD

**Discussion by simkimsia** [executablebooks/meta#712] [Q&A] [answered]
I am looking at https://github.com/executablebooks/jupyter-book/blob/master/pyproject.toml#L47 master branch and i saw that it's using sphinx-book-theme ~= 0.1.4

Given that sphinx-book-theme is now at 0.3.2 and i intend to submit a PR to sphinx book theme, how do I still use jupyter-book latest version so that it can also use the latest sphinx-book-theme?

or for that matter if i forked sphinx-book-theme how do i make jupyter-book use my forked version?

**Comment by choldgraf** [comment, marked as answer]
We're intending to update the version quite soon, there's a PR for this here: https://github.com/executablebooks/jupyter-book/pull/1679

**Reply by simkimsia** [reply]
Thank you for this. I will subscribe to that PR
